cross/ncursesw/PLIST

```diff
@@ -10,19 +10,16 @@ bin:bin/tput
 bin:bin/tset
 lnk:lib/libformw.so
 lnk:lib/libformw.so.6
-lib:lib/libformw.so.6.4
+lib:lib/libformw.so.6.5
 lnk:lib/libmenuw.so
 lnk:lib/libmenuw.so.6
-lib:lib/libmenuw.so.6.4
+lib:lib/libmenuw.so.6.5
 lnk:lib/libncursesw.so
 lnk:lib/libncursesw.so.6
-lib:lib/libncursesw.so.6.4
+lib:lib/libncursesw.so.6.5
 lnk:lib/libpanelw.so
 lnk:lib/libpanelw.so.6
-lib:lib/libpanelw.so.6.4
-lnk:lib/libtinfow.so
-lnk:lib/libtinfow.so.6
-lib:lib/libtinfow.so.6.4
```

Comment:
we need `libtinfow` with the definition of `NCURSES_WITH_TINFO=yes`
_will be omitted if not defined_

Reply:
Gotcha, fix now already in my local git, pending python fix for the issue I cannot reproduce ... sigh.